kore/src/Kore/Repl/Data.hs

```diff
@@ -351,8 +351,9 @@ helpText =
     \                                      these scopes are used for filtering\
                                            \ the logged information, for example,\
                                            \ '[]' will log all scopes\n\
-    \                                      <type> can be 'stderr' or\n\
-                                           \'file filename'\n\
+    \                                      <type> can be 'stderr' or 'filetext';\
+                                           \ the latter prints the output to\
+                                           \ \"./kore-(date).log\"\n\
```

Comment:
Is this correct, that `kore-repl` has a hard-coded filename for the log? What about `kore-exec`?

Reply:
It shouldn't be hard-coded in `kore-repl`